minidump-processor/src/stackwalker/mod.rs

```diff
@@ -206,13 +222,10 @@ where
         let mut maybe_frame = Some(StackFrame::from_context(ctx, FrameTrust::Context));
         while let Some(mut frame) = maybe_frame {
             fill_source_line_info(&mut frame, modules, symbol_provider).await;
-            trace!(
-                "unwinding {}",
-                frame
-                    .function_name
-                    .clone()
-                    .unwrap_or_else(|| format!("0x{:08x}", frame.instruction))
-            );
+            match frame.function_name.as_ref() {
+                Some(name) => trace!("unwinding {}", name),
+                None => trace!("unwinding 0x{:016x}", frame.instruction),
+            }
```

Comment:
I was wondering, not sure how these trace macros internally work, if any code is called when you are not subscribed to this log level. Since this is doing string cloning and allocation, I tried to avoid that.